db/seeds.rb

```diff
@@ -189,11 +189,13 @@
   ]
 )
 
-# Enablers/Barriers
-EnablerBarrier.create(
+# Statuses for DeliveryUpdates
+DeliveryUpdateStatus.create(
   [
-    { name: "Enablers/barriers to study delivery" },
-    { name: "Enablers/barriers to dissemination or uptake of results" },
-    { name: "Enablers/barriers to patient recruitment" },
+    { name: "Progressing fine", good_medium_bad_or_neutral: "good" },
+    { name: "Not started", good_medium_bad_or_neutral: "neutral" },
+    { name: "Minor problems or delays", good_medium_bad_or_neutral: "medium" },
+    { name: "Major problems or delays", good_medium_bad_or_neutral: "bad" },
+    { name: "Completed", good_medium_bad_or_neutral: "neutral" }
```

Comment:
@stevenday – Are you sure that "completed" should be "neutral"? Is "completed" not a "good" thing?